Aardvark/ARKEmailBugReportConfiguration.m

```diff
@@ -0,0 +1,58 @@
+//
+//  ARKEmailBugReportConfiguration.m
+//  Aardvark
+//
+//  Created by Nick Entin on 4/14/18.
+//  Copyright 2018 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "ARKEmailBugReportConfiguration.h"
+#import "ARKEmailBugReportConfiguration_Protected.h"
+
+
+@interface ARKEmailBugReportConfiguration ()
+
+@property (nonatomic, readwrite) BOOL includesScreenshot;
+@property (nonatomic, readwrite) BOOL includesViewHierarchyDescription;
+
+@end
+
+
+@implementation ARKEmailBugReportConfiguration
+
+- (instancetype)initWithScreenshot:(BOOL)includesScreenshot viewHierarchyDescription:(BOOL)includesViewHierarchyDescription;
+{
+    self = [super init];
+    
+    _prefilledEmailSubject = @"";
```

Comment:
it may be worth having a quick comment here stating: "If any of the below default values change, remember to update the header file"

That may also be overkill. I leave it up to you.

Reply:
I feel like making behavioral changes like changing default values should imply needing to update the header docs, and if it doesn't then you don't have enough header docs.  Maybe that's just optimistic thinking, but adding comments saying that things are documented feels backwards.

Reply:
Works for me!